genai-perf/genai_perf/types.py

```diff
@@ -38,6 +38,7 @@
 GpuRecords: TypeAlias = Dict[GpuId, TelemetryRecords]
 PerfRecords: TypeAlias = Dict[str, "Record"]  # type: ignore
 PerfMetricName: TypeAlias = str
+RecordValue = Union[float, int]
```

Comment:
Could we make this to be called something more generic (e.g. `Number`). Asking since I could see this type being useful outside of the `Record` class.

Reply:
There's no restriction on having multiple types alias to the same thing. In general, I feel that it defeats the purpose of typing to make them generic. If something else (that's not a record) also has this type they should create their own type. This both serves as documentation and can prevent bugs from being introduced.